sale_order_type/models/account_invoice.py

```diff
@@ -3,12 +3,15 @@
 # For copyright and license notices, see __openerp__.py file in root directory
 ##############################################################################
 
-from openerp import api, models
+from openerp import api, models, fields
 
 
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
 
+    sale_type_id = fields.Many2one(
```

Comment:
@pedrobaeza, journal contains accounting specific informations. ```Sale Type``` is more like a ```Line of business```. That is, for the same journal, I could configure several lines of business: I should not be forced to create a journal for every line of business